templates/2.0/agent.j2

```diff
@@ -19,7 +19,7 @@ version = "{{ agent.version }}"
 uuid = "{{ agent.uuid }}"
 
 # The binding address and port for the agent server
-ip = "{{ agent.ip }}"
+ip = {{ agent.ip }}
```

Comment:
No needed change. The correct one is with quotes, but there is code to add them if they are missing. But I see why is there, to match the rest of the templates.